docs/specs/apollo_3.0_development_kits_user_manual_cn.md

```diff
@@ -569,32 +570,56 @@ rostopic echo /apollo/sensor/camera/traffic/image_short
 ![图片](images/radar_parameter_value.png)
 	
 
-## 激光雷达安装与配置
-
-![图片](images/lidar_look.jpeg)
+## 激光雷达安装与数据验证
+ - 激光雷达型号：80-VLP-16（velodyne 16线激光雷达）
+ - 更多详细参数可参考：[https://velodynelidar.com/vlp-16.html](https://velodynelidar.com/vlp-16.html)
+ 
+![图片](images/lidar_look.png)
 
-**激光雷达型号**：80-VLP-16（velodyne 16线激光雷达）
+### 激光雷达的安装固定
+ - 16线激光雷达及接口盒要牢靠固定在车身上，连接到接口盒上的接头要牢靠接插。水平安装在车顶部，对地高度1.5米，水平放置，精度在2度以内。连接线缆带屏蔽，接口镀金， 网线两头需贴上标签注明设备名称。安装位置如下图：
+ 
+![图片](images/lidar_installation_position.png)
 
-**更多详细参数可参考**：[https://velodynelidar.com/vlp-16.html](https://velodynelidar.com/vlp-16.html)
+ - 安装激光雷达时线缆方向朝向车辆的后方。
 
-安装要求：16线激光雷达及控制盒要牢靠固定在车身上，连接到控制盒上的接头要牢靠接插。水平安装在车顶部，对地高度1.5米，水平放置，精度在2度以内。连接线缆带屏蔽，接口镀金，可参考京东线缆购买网址：[https://item.jd.com/1342004.html](https://item.jd.com/1342004.html) 网线两头需贴上标签注明设备名称。安装位置如下图：
+### 激光雷达与车辆的接线
 
-![图片](images/lidar_install_position.png)
+ - 激光雷达的接口盒有三个接口，分别为数据口(以太网接口)、12V电源接口、GPS授时接口，如下图所示：
+ 
+![图片](images/lidar_socket.jpeg)
 
-**设备连接说明**：数据输出端口通过以太网连接到工控机以太网口。
+ - 数据口：通过以太网线缆将接口盒连接到IPC。
+ - 12V电源接口：可以使用雷达自带的12V适配器进行供电或者使用自己的电源线连接12V电源供电。
+ - GPS授时接口：需要将GPS的授时接口与激光雷达接口盒的授时接口进行连接。
+ 
+### 激光雷达的配置及启动
 
-控制盒接口如图所示：
+ - 设置激光雷达ip和IPC的ip处在相同号段：雷达出厂默认ip地址为192.168.1.201，将IPC的ip地址修改为静态ip且与雷达ip地址处于相同的号段，例如192.168.1.xxx，如下图所示：
```

Comment:
I thought our practice is to reset the lidar ip to be the same range of IPC ip instead of doing it opposite way? Correct me if I ma wrong.

Reply:
I have modified it already, please review it again. think you !